quarkus/config-api/src/main/java/org/keycloak/config/ProxyOptions.java

```diff
@@ -19,4 +19,14 @@ public enum Mode {
             .category(OptionCategory.PROXY)
             .defaultValue(Boolean.FALSE)
             .build();
+
+    public static final Option<Boolean> PROXY_FORWARDED_HEADER_ENABLED = new OptionBuilder<>("proxy-allow-forwarded-header", Boolean.class)
+            .category(OptionCategory.PROXY)
+            .defaultValue(Boolean.FALSE)
+            .build();
+
+    public static final Option<Boolean> PROXY_X_FORWARDED_HEADER_ENABLED = new OptionBuilder<>("proxy-allow-x-forwarded-header", Boolean.class)
+            .category(OptionCategory.PROXY)
+            .defaultValue(Boolean.FALSE)
+            .build();
```

Comment:
Should we maybe add something around these two new options to the Proxy guide?

Reply:
The options are not supported and not shown in any of our documentation (e..g: help). 

But I think it would be nice to document the new behavior in the release notes and upgrade guides. People should be aware that now proxies must take the standard `Forwarded` header into account when overriding headers.

Reply:
If it's not supported, why do we need a KC config wrapper for it? :)

Reply:
Unfortunately, our configuration requires a 1:N mapping, even though the KC option is not exposed.